gnosis/cowswap
# profile page && styled-components grid

# Summary
Fixes #988 

*High-level description of what your changes are accomplishing*
Profile view implemented
Custom common components created in order to have less small components and avoid duplication of code. Check it please, maybe we could use it somehow or find a better decision?

  # To Test

1. Visit profile page, please notice: all data are mocked
2. as new global styles were applied, it need testing to
